src/tiled/newtilesetdialog.cpp

```diff
@@ -284,6 +282,11 @@ void NewTilesetDialog::updateOkButton()
  */
 void NewTilesetDialog::pickColorFromImage()
 {
+    if (mPopup != nullptr) {
```

Comment:
On second thought, you probably did test it, but it worked by chance because `mPopup` isn't being initialized to `nullptr`.